src/domain/reservation/manage/filters/ManageReservationsFilters.js

```diff
@@ -107,75 +112,73 @@ class ManageReservationsFilters extends React.Component {
       <div className="app-ManageReservationsFilters">
         <Grid>
           <Row>
```

Comment:
This file change basically moving the a big `Row` to 2 smaller `Row`. HTML change, mostly useless nothing to review here